tests/acceptance/features/webUIUpload/upload.feature

```diff
@@ -63,35 +65,35 @@ Feature: File Upload
 
   @smokeTest @ocisSmokeTest
   Scenario: uploading a big file (when chunking is implemented this upload should be chunked)
-    Given a file with the size of "30000000" bytes and the name "big-video.mp4" has been created locally
+    Given a file with the size of "30000000" bytes and the name "big-video.mp4" has been created locally in the server
```

Comment:
This step also reds a bit strange - "created locally in the server"

For steps that are done "in the middleware" but where the middleware does not do anything "in/on/to the server", we could improve the words. Maybe we can end steps with "in the middleware"?

So this step would end with "has been created locally in the middleware".

And we can easily have the test code match steps that end in "in the middleware"

Reply:
yes, @phil-davis, the server is not involved at all, so refactorings should be made. I've created an issue for this https://github.com/owncloud/owncloud-test-middleware/issues/96